py-polars/polars/dataframe/frame.py

```diff
@@ -1791,13 +1789,18 @@ def item(self, row: int | None = None, column: int | str | None = None) -> Any:
             return self._df.select_at_idx(0).get_idx(0)
 
         elif row is None or column is None:
-            raise ValueError("Cannot call '.item()' with only one of 'row' or 'column'")
+            raise ValueError(
+                "Cannot call '.item()' with only one of 'row' or 'column'."
```

Comment:
I'd like to have the following style guidelines for error messages:

* First word is lowercase
* If the message is a single sentence, no period
* If the message is multiple sentences, add a period and start new sentences with a capital letter
* Code snippets enclosed in backticks.
* When adding format strings, use the `!r` representation, e.g. `f"Column name: {name!r}"`

See for reference:
https://github.com/pola-rs/polars/pull/7362#issuecomment-1456288951

```suggestion
                "cannot call `.item()` with only one of `row` or `column`"
```

Reply:
I was thinking much the same! Do we have somewhere to consolidate information on the "house style"?

Reply:
Actually, after re-reading the post I linked to, I would update my 'guidelines' a bit. The 'main' error message is lowercase no period. Any further explanatory comments should be after a newline and they will be properly capitalized/punctuated.

I think that would be nicely readable and consistent. For example, this is a standard Python error message:
`IndexError: list index out of range`

Reply:
Regarding where to put this information, I'll add a code style / guidelines section to the contributing guide.

Reply:
Already made a tiny start here: https://github.com/pola-rs/polars/tree/main/contribution#readme

Reply:
Good stuff! (I'll fix the lint on this in a few minutes).